src/Lib.hs

```diff
@@ -315,8 +326,15 @@ runUDCCommand command absoluteRomefilePath verbose romeVersion = do
       <> romeVersionToString vers
       <> noColorControlSequence
 
+  -- case (mS3BucketName, mEnginePath) of
```

Comment:
This is not needed at all I think, it was probably an initial sketch leftover?

Reply:
yep, not needed. I was mapping all possible combinations.